pom.xml

```diff
@@ -116,9 +122,11 @@
 	</dependencyManagement>
 
 	<properties>
-		<openMRSVersion>1.9.9</openMRSVersion>
+		<openMRSVersion>2.0.0</openMRSVersion>
 		<log4jScope>provided</log4jScope>
 		<commonsLangScope>provided</commonsLangScope>
+        <javaCompilerVersion>1.8</javaCompilerVersion>
+        <legacyUiVersion>0.1</legacyUiVersion>
```

Comment:
Is there any reason why you are not using the latest version of the legacyui module?

Reply:
Based on our past practice, we always require the minimum supported version that way an implementation doesn't have to necessary have to run the latest legacyui version in order to use the latest version of patient matching with the new changes.